src/kirby/components/loading-overlay/loading-overlay.component.tns.html

```diff
@@ -0,0 +1,18 @@
+<GridLayout #overlayWrapperRef>
+    <StackLayout class="content-wrapper">
+        <ng-content></ng-content>
+    </StackLayout>
+    <FlexboxLayout
```

Comment:
Is a `FlexboxLayout` really needed here?

Without testing it, i actually think you can do like:

```html
<GridLayout
  *ngIf="isLoading"
  [class.backdrop]="showBackdrop"
  class="spinner-wrapper"
  columns="*" 
  rows="*" 
  horizontalAlignment="center">
  <kirby-spinner></kirby-spinner>
</GridLayout>
```

We should try to avoid `FlexboxLayout` where we can, as is does have a performance hit.

Reply:
I just learned this. Will try to do it with gridlayout